src/App.tsx

```diff
@@ -748,7 +703,11 @@ export default function App() {
   return (
     <StyledEngineProvider injectFirst>
       <CssBaseline />
-      <UserProvider data={{ user, reloadUser }}>{getContent()}</UserProvider>
+      <UserProvider data={{ user, reloadUser }}>
+        <OrganizationProvider data={{ selectedOrganization, setSelectedOrganization, organizations, reloadData }}>
```

Comment:
for the typescript error, we could do a 
```
(selectedOrganization && (
   ...
)
```
on line 704.

Feels like our App.tsx could use some cleanup at some point. We added a lot of patch up style code recently.

Reply:
ooh, that's nice. it'd allow me to get rid of a whole bunch of `selectedOrganization &&`